app/models/CSVModels.js

```diff
@@ -0,0 +1,26 @@
+import csv from 'csvtojson';
+import mongoose from '../services/mongoose.js';
+
+const name = 'inspiration'
+
+// using the one csv in the repo for now.
+// later, let this be variable.
+const file_path = `./assets/decks/${name}/${name}.csv`
+
+export const rows = await csv().fromFile(file_path);
+
+const schema_data = {}
+
+for (var header in rows[0]) {
```

Comment:
This should be `for const` if the value isn't changing. I believe `eslint` should have picked up on this. Run `yarn lint` in your terminal, or install an `eslint` plugin for your editor to see linting errors inline.